mllib/src/main/scala/org/apache/spark/mllib/api/python/PythonMLLibAPI.scala

```diff
@@ -407,6 +406,35 @@ private[python] class PythonMLLibAPI extends Serializable {
   }
 
   /**
+   * A Wrapper of FPGrowthModel to provide helper method for Python
+   */
+  private[python] class FPGrowthModelWrapper(model: FPGrowthModel[Any])
```

Comment:
It might be simpler to move this out of `class PythonMLLibAPI` to use with py4j. See #5243.